code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1179,7 +1184,7 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 
 
 
-/datum/species/proc/harm(mob/living/carbon/human/user, mob/living/carbon/human/target, datum/martial_art/attacker_style)
+/datum/species/proc/harm(mob/living/carbon/human/user, mob/living/carbon/human/target, datum/martial_art/attacker_style) //the datum belongs to target, fuck you
```

Comment:
This species datum is actually owned by the target for some reason. 
on_harm is owned by the actual attacker